### PR TITLE
add cold backup and restore scripts

### PR DESCRIPTION
**Pull request by jeromeleonard**

### CODE REVIEW

File: prod1-cortex/README.md [hunk 1]
```diff
@@ -191,3 +191,7 @@ bash ./scripts/reset.sh
 Run the *init.sh* script to reload a new *production* instance. 
  
 
+### Backup / Restore
+
+This profile includes two utility scripts to assist with performing cold backups and restores. You can find these scripts here: [./scripts/backup.sh](./scripts/backup.sh) and [./scripts/restore.sh](./scripts/restore.sh)
+For detailed information about backup and restore strategies and processes, please refer to the [dedicated documentation](https://docs.strangebee.com/thehive/operations/backup-restore/overview/).
```
Comment (undebutant) [inline prod1-cortex/README.md:197]:
```suggestion
This profile includes two utility scripts to assist with performing cold backups and restores. You can find these scripts here: [./scripts/backup.sh](./scripts/backup.sh) and [./scripts/restore.sh](./scripts/restore.sh).
For detailed information about backup and restore strategies and processes, please refer to the [dedicated documentation](https://docs.strangebee.com/thehive/operations/backup-restore/#overview).
```
Fix broken link

File: prod1-cortex/scripts/backup.sh [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+## - Do not modify the rest of the script unless necessary.
+##
+## ============================================================
+## DO NOT MODIFY ANYTHING BELOW THIS LINE
+## ============================================================
+
+# Display help message
+if [[ "$1" == "--help" || "$1" == "-h" ]]
+then
+  echo "Usage: $0 [BACKUP_ROOT_FOLDER]"
```
Comment (undebutant) [inline prod1-cortex/scripts/backup.sh:50]:
```suggestion
  echo "Usage: $0 [DOCKER_COMPOSE_PATH] [BACKUP_ROOT_FOLDER]"
```

File: prod1-cortex/scripts/backup.sh [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  echo "                      If not provided, you will be prompted for a folder, with a default of './backup'."
+  echo
+  echo "Examples:"
+  echo "  $0 /path/to/docker-compose-folder /path/to/backup  Perform backup with specified root folder."
+  echo "  $0                 Prompt for docker compose folder and backup root folder."
+  exit 0
+fi
+
+## Checks if the first argument is provided.
+## If it is, the script uses it as the value for BACKUP_ROOT_FOLDER
```
Comment (undebutant) [inline prod1-cortex/scripts/backup.sh:67]:
```suggestion
## If it is, the script uses it as the value for DOCKER_COMPOSE_PATH
```

File: prod1-cortex/scripts/backup.sh [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+## Copy Nginx certificates
+echo "Starting backup of Nginx and certificates..."
+rsync -aW --no-compress ${DOCKER_COMPOSE_PATH}/nginx/ ${BACKUP_FOLDER}/nginx || { echo " Backup of Nginx failed"; exit 1; }
+rsync -aW --no-compress ${DOCKER_COMPOSE_PATH}/certificates/ ${BACKUP_FOLDER}/certificates || { echo " Backup of Nginx and certificates failed"; exit 1; }
+echo "Backup of certificates completed."
+
+## Restart services
+echo "Restarting services..."
+docker compose up -d
```
Comment (undebutant) [inline prod1-cortex/scripts/backup.sh:134]:
```suggestion
docker compose up -d -f ${DOCKER_COMPOSE_PATH}/docker-compose.yml
```

File: prod1-cortex/scripts/restore.sh [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+  echo "                      If not provided, you will be prompted for a folder or exit; no default folder is used."
+  echo
+  echo "Examples:"
+  echo "  $0 /path/to/docker-compose-folder /path/to/backup-folder  restores backup stored in the specified folder."
+  echo "  $0                 Prompt for docker compose folder and backup folder to restore."
+  exit 0
+fi
+
+## Checks if the first argument is provided.
+## If it is, the script uses it as the value for BACKUP_ROOT_FOLDER
```
Comment (undebutant) [inline prod1-cortex/scripts/restore.sh:68]:
```suggestion
## If it is, the script uses it as the value for DOCKER_COMPOSE_PATH
```

File: prod1-thehive/scripts/restore.sh [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+# Copy Nginx certificates
+echo "Restoring Nginx data and configuration..."
+rsync -a ${BACKUP_FOLDER}/nginx/ ${DOCKER_COMPOSE_PATH}/nginx  ||
+ { echo " Nginx configuration and certificates restore failed"; exit 1; } 
+rsync -a ${BACKUP_FOLDER}/certificates/ ${DOCKER_COMPOSE_PATH}/certificates  ||
+ { echo " certificates restore failed"; exit 1; } 
+
+## Restart services
+echo "Restarting services..."
+docker compose up -d
```
Comment (undebutant) [inline prod1-thehive/scripts/restore.sh:134]:
```suggestion
docker compose up -d -f ${DOCKER_COMPOSE_PATH}/docker-compose.yml
```

**Review by undebutant — COMMENTED** [review]
I suggested some changes for comments and commands.

I only created the suggestion for `prod1-cortex` folder, but they should be applied to all 5 profiles.

File: prod1-cortex/README.md (hunk 1 above)
Comment (jeromeleonard) [reply prod1-cortex/README.md:197]: I anticipated my updates on docs.strangebee.com

**Pull request merged**